deepchem/utils/differentiation_utils/editable_module.py

```diff
@@ -0,0 +1,502 @@
+"""
+Editable Utils
+Derived From: https://github.com/xitorch/xitorch/blob/master/xitorch/_core/editable_module.py
+"""
+import inspect
+import warnings
+from abc import abstractmethod
+import copy
+import torch
+from typing import Sequence, Union, Dict, List
+from deepchem.utils.attribute_utils import get_attr, set_attr, del_attr
+
+__all__ = ["EditableModule"]
+
+torch_float_type = [torch.float32, torch.float, torch.float64, torch.float16]
+
+
+class EditableModule(object):
```

Comment:
Add editable module to the docs

Reply:
Its added already, in the `utils.rst` file.
Do you mean that I should create another section for it.